web/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
Auto-generating this code so that it stays up to date sounds way better than this hack. 

However, is it worth sticking this in for the meantime?